core-ui/src/state/navigation/categories.ts

```diff
@@ -0,0 +1,70 @@
+import { NavNode } from '../types';
+
+export enum PredefinedCategories {
+  'workloads' = 'Workloads',
+  'discovery-and-network' = 'Discovery-and-network',
+  'istio' = 'Istio',
+  'service-management' = 'Service-management',
+  'storage' = 'Storage',
+  'apps' = 'Apps',
+  'configuration' = 'Configuration',
+  'integration' = 'Integration',
+}
```

Comment:
Recently I've found an interesting [article](https://frontlive.pl/blog/typescript-enum) (in Polish) about enums. Have you considered replacing them with Objects like in last example in the link above?